### PR TITLE
test: Make sure basic sessions tests run

### PR DESCRIPTION
**Pull request by bruno-garcia**
Tests were not part of the pbxproj hence not running. Updated tests following last changes to closing sessions. 

No change on the actual sessions logic was needed, just updated the tests.

Note: Please discuss/review and merge #507 before this PR.

### CODE REVIEW

File: Samples/iOS-ObjectiveC/iOS-ObjectiveC/ViewController.m [hunk 1]
```diff
@@ -40,7 +40,7 @@ - (IBAction)addBreadcrumb:(id)sender {
 
 - (IBAction)captureMessage:(id)sender {
     NSString *eventId = [SentrySDK captureMessage:@"Yeah captured a message"];
-    // Returns eventId in case of successfull processed event
+    // Returns eventId in case of successful processed event
```
Comment (philipphofmann) [inline Samples/iOS-ObjectiveC/iOS-ObjectiveC/ViewController.m:43]: Nice typo fixing. 🎉

File: Sources/Sentry/SentryHub.m [hunk 2]
```diff
@@ -83,14 +83,15 @@ - (void)closeCachedSession {
     SentryFileManager *fileManager = [[self getClient] fileManager];
     SentrySession *session = [fileManager readCurrentSession];
     if (nil != session) {
+        [SentryLog logWithMessage:[NSString stringWithFormat:@"A cached session was found."] andLevel:kSentryLogLevelDebug];
```
Comment (philipphofmann) [inline Sources/Sentry/SentryHub.m:86]: `l`: I think we don't need the `stringWithFormat` here. The same applies for the other logs in this method.
```suggestion
        [SentryLog logWithMessage:@"A cached session was found." andLevel:kSentryLogLevelDebug];
```

Comment (philipphofmann) [inline Sources/Sentry/SentryHub.m:86]: `l`: What about prefixing all the log messages here with `closeCachedSession` or something similar. The log message itself doesn't tell me this is happening during closing a session.

File: Sources/Sentry/SentrySessionTracker.m [hunk 3]
```diff
@@ -84,8 +84,8 @@ - (void)willResignActive {
 }
 
 - (void)willTerminate {
-    NSDate *from = nil == self.lastInForeground ? [NSDate date] : self.lastInForeground;
-    [[SentrySDK currentHub] endSessionWithTimestamp:from];
+    NSDate *sessionEnded = nil == self.lastInForeground ? [NSDate date] : self.lastInForeground;
```
Comment (philipphofmann) [inline Sources/Sentry/SentrySessionTracker.m:87]: The name for this is clearer for me now 👏

File: Tests/SentryTests/Networking/SentryHttpDateParserTests.swift [hunk 4]
```diff
@@ -5,7 +5,6 @@ class SentryHttpDateParserTests: XCTestCase {
     
     private var currentDateProvider: TestCurrentDateProvider!
     private var sut: HttpDateParser!
-
```
Comment (philipphofmann) [inline Tests/SentryTests/Networking/SentryHttpDateParserTests.swift:8]: Awesome 💯

File: Sources/Sentry/SentryHub.m (hunk 2 above)
Comment (bruno-garcia) [reply Sources/Sentry/SentryHub.m]: Logging is generally suboptimal, I usually just search for the string if I want to get to that line of code. We could pass a "category" to a logger instance 

>The log message itself doesn't tell me this is happening during closing a session.

The log message is just saying it found a cached file

**Pull request merged**